### ISSUE TITLE
Date filters do not give specific enough error message

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/datagateway#952]
**Description**:
Date filters do not give specific enough error message

- Adding something similar to the image below should be sufficient 
![Screenshot 2021-11-26 111131](https://user-images.githubusercontent.com/83226114/143572409-c97630a3-80dd-40bd-b08b-960e7e927bc2.png)

**Acceptance criteria**:
- [x] Satisfy WCAG 2.1 3.3.1 and 3.3.3
